Configuration/StandardSequences/python/AlCaRecoStreams_cff.py

```diff
@@ -197,6 +200,7 @@
 pathALCARECOTkAlCosmicsCosmicTF0THLT = cms.Path(seqALCARECOTkAlCosmicsCosmicTF0THLT*ALCARECOTkAlCosmicsCosmicTF0TDQM)
 pathALCARECOTkAlCosmicsRegional0THLT = cms.Path(seqALCARECOTkAlCosmicsRegional0THLT*ALCARECOTkAlCosmicsRegional0TDQM)
 pathALCARECOMuAlGlobalCosmicsInCollisions = cms.Path(seqALCARECOMuAlGlobalCosmicsInCollisions*ALCARECOMuAlGlobalCosmicsInCollisionsDQM)
+pathALCARECOMuAlGlobalCosmicsInCollisionsTracks = cms.Path(seqALCARECOMuAlGlobalCosmicsInCollisionsGeneralTracks)
```

Comment:
@lpernie you should remove this line as well, otherwise:
```
Step: ALCA Spec: ['MuAlGlobalCosmics', 'DtCalib']
DL enriching full MuAlGlobalCosmics+DtCalib
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-08-11-2300/bin/slc6_amd64_gcc530/cmsDriver.py", line 55, in <module>
    run()
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-08-11-2300/bin/slc6_amd64_gcc530/cmsDriver.py", line 27, in run
    configBuilder.prepare()
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-08-11-2300/python/Configuration/Applications/ConfigBuilder.py", line 2058, in prepare
    self.addStandardSequences()
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-08-11-2300/python/Configuration/Applications/ConfigBuilder.py", line 743, in addStandardSequences
    getattr(self,"prepare_"+stepName)(sequence = '+'.join(stepSpec))
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-08-11-2300/python/Configuration/Applications/ConfigBuilder.py", line 1236, in prepare_ALCA
    alcaConfig=self.loadDefaultOrSpecifiedCFF(sequence,self.ALCADefaultCFF)
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-08-11-2300/python/Configuration/Applications/ConfigBuilder.py", line 1188, in loadDefaultOrSpecifiedCFF
    l=self.loadAndRemember(defaultCFF,unsch)
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-08-11-2300/python/Configuration/Applications/ConfigBuilder.py", line 323, in loadAndRemember
    self.process.load(includeFile)
  File "/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-08-11-2300/python/FWCore/ParameterSet/Config.py", line 511, in load
    module = __import__(moduleName)
  File "/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-08-11-2300/python/Configuration/StandardSequences/AlCaRecoStreams_cff.py", line 203, in <module>
    pathALCARECOMuAlGlobalCosmicsInCollisionsTracks = cms.Path(seqALCARECOMuAlGlobalCosmicsInCollisionsGeneralTracks)
NameError: name 'seqALCARECOMuAlGlobalCosmicsInCollisionsGeneralTracks' is not defined
```